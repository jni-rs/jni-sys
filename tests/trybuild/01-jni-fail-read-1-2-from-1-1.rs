
use jni_sys::JNINativeInterface_;

pub fn main() {
    unsafe {
        let jni = std::mem::zeroed::<JNINativeInterface_>();
        let _1_2_function = jni.v1_1.FromReflectedMethod;
    }
}
