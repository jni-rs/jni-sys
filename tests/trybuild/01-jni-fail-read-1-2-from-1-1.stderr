error[E0609]: no field `FromReflectedMethod` on type `JNINativeInterface__1_1`
 --> tests/trybuild/01-jni-fail-read-1-2-from-1-1.rs:7:38
  |
7 |         let _1_2_function = jni.v1_1.FromReflectedMethod;
  |                                      ^^^^^^^^^^^^^^^^^^^ unknown field
  |
  = note: available fields are: `GetVersion`, `DefineClass`, `FindClass`, `GetSuperclass`, `IsAssignableFrom` ... and 203 others
